study-builder/studies/osteo/study.conf

```diff
@@ -2606,43 +2647,174 @@
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "THERAPIES_RECEIVED"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_EVER_RELAPSED"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "EVER_RELAPSED"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_CURRENTLY_TREATED"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "CURRENTLY_TREATED"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_OTHER_CANCERS"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "OTHER_CANCERS"
+            }
+          },
+          {
+            # composite parent: CHILD_OTHER_CANCERS_LIST -> OTHER_CANCERS_LIST
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_OTHER_CANCER_NAME"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "OTHER_CANCER_NAME"
+            }
+          },
+          {
+            # composite parent: CHILD_OTHER_CANCERS_LIST -> OTHER_CANCERS_LIST
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_OTHER_CANCER_YEAR"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "OTHER_CANCER_YEAR"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_EXPERIENCE"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "EXPERIENCE"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_HISPANIC"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "HISPANIC"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_RACE"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "RACE"
+            }
+          },
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CHILD_HOW_HEAR"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "HOW_HEAR"
+            }
+          },
+        ]
       },
+      "cancelExpr": """!user.studies["CMI-OSTEO"].forms["ABOUTCHILD"].hasInstance()""",
```

Comment:
wondering if we should add ["ABOUTCHILD"] status = "Completed" for additional safety !!

Reply:
I think that might result in the wrong behavior. User is not required to submit the `about_child` activity, so it is likely that when we get there during age-up, the `about_child` is not complete yet. Regardless of whether it's complete or not, it's still safe to do copying -- if there's no answer, we don't copy anything over.